lib/cli.js

```diff
@@ -47,7 +47,7 @@ module.exports = function(program) {
      * Trying to load config.
      * Custom config path can be specified using '-c' option.
      */
-    if (!config && !program.preset && !program.autoConfigure) {
+    if (!config && !program.preset && !program.autoConfigure && !program.ignoreMissingConfig) {
```

Comment:
Shouldn't we just exit(0) on `program.ignoreMissingConfig` without a config? Like...
```js
if (program.ignoreMissingConfig) {
  defer.resolve();
  return returnArgs;
}
```